### DISCUSSION TITLE
Issue tracker is closed until further notice.

### DISCUSSION THREAD

**Discussion by wereturtle** [KDE/ghostwriter#642] [Announcements]
I apologize for any inconvenience.  The closing of the issue tracker does NOT mean ghostwriter is dead.  I just simply need some peace and quiet from the trolls.  I have opened the new Discussions feature for the community to help each other out.

@kimcosmos Regarding autosave needing a warning...  Perhaps you would like to harass the gedit maintainers for having the exact same option with no warning to give your file a name first?  And yes, if you fail to give your file a name, it will NOT actually autosave.

![image](https://user-images.githubusercontent.com/10375170/126446859-fd4840f2-9b7c-4c8a-a47b-e0868b87989e.png)

Everyone, please feel free to thank @kimcosmos for being the straw that broke the camel's back.

**Comment by npanzica-qontigo** [comment]
I can't get the discussion feature to work. It won't let me choose a category and without that I cannot post my question.

**Reply by abenson** [reply]
Can you describe the issue further? Does it give an error or just not accept your click?

**Comment by npanzica-qontigo** [comment]
I'm sorry I found the solution right after I wrote this. (I hate it when that happens..) Didn't mean to waste your time. This can be closed. I think I was clicking on things in the wrong order.  I will try my question now.

**Comment by npanzica-qontigo** [comment]
OK, it's still not working. I thought it was fixed. It is not accepting my click. 

I can write the message out. But when I try to post it, I can't because I have to choose a category. When I try to do so/click the category button, nothing happens.

**Reply by abenson** [reply]
What browser are you using?  It seems to work fine for me.

**Comment by npanzica-qontigo** [comment]
Thank you! I think it is my browser. I use a very old version of Firefox because it retains a feature I value. But because it's so old, sometimes elements on modern web pages do not display or work right. I will try this in Chrome, and I have a feeling you are right as I encounter this sort of thing on other sites.

**Comment by npanzica-qontigo** [comment]
Yes, it was the browser.  I can select a category easily in Chrome.

**Comment by linuxguy123** [comment]
Is issue tracker still closed ?

Is this the "real" code repository for Ghostwriter or is there another one in KDEland somewhere ?
